### PR TITLE
Added Scala support for Gob's Program.

### PR DESCRIPTION
**Pull request by mpancia**

### CODE REVIEW

File: gobsprogram.scala [hunk 1]
```diff
@@ -0,0 +1,8 @@
+object GobsProgram { 
+	def main(args: Array[String]): Unit = {
+		print("\033[1;32;40m")
+		println("Gob's Program: Y/N?")
+		val doGob = readLine("?")
+		if(doGob == "Y") { while(true) println("PENUS PENUS PENUS PENUS PENUS") }
```
Comment (cswingler) [inline gobsprogram.scala]: I believe it's pronounced "Penus"

**Comment by anthonypants** [comment]
Please remember that due to the [Program Specification](https://github.com/jabberzac/gobsprogram/wiki/Gob%27s-Program-Specification) there must be a single space following the question mark, and the output loop must contain double-spaces between words.

**Comment by anthonypants** [comment]
Also a double-space between "Gob's Program:" and "Y/N?"

**Comment by mpancia** [comment]
@anthonypants Sorry, I didn't realize there was a program specification (duh). That's what I get for blindly following github links. New PR ought to be okay. 👍

File: gobsprogram.scala [hunk 2]
```diff
@@ -0,0 +1,8 @@
+object GobsProgram { 
+	def main(args: Array[String]): Unit = {
+		print("\033[1;32;40m")
+		println("Gob's Program:  Y/N?")
```
Comment (mpancia) [inline gobsprogram.scala]: Added double space between colon and Y.

File: gobsprogram.scala [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+		val doGob = readLine("? ")
+		if(doGob.toLowerCase == "y") { while(true) print("Penus  ") }
```
Comment (mpancia) [inline gobsprogram.scala]: Added lowercase support, added double spacing between `Penus`es.

**Pull request merged**